controller_manager/test/test_controller_manager_srvs.cpp

```diff
@@ -118,21 +118,6 @@ TEST_F(TestControllerManagerSrvs, list_controller_types)
   ASSERT_THAT(
     result->base_classes,
     ::testing::Contains("controller_interface::ControllerInterface"));
-
-  // std::shared_ptr<ControllerLoaderMock> mock_loader(new ControllerLoaderMock);
```

Comment:
Can we delete the actual Mock as well? 
https://github.com/ros-controls/ros2_control/pull/175/files#diff-5cd042aad8bad76fffe9ac45c0da83f0R122

Reply:
I missed this one. Removed in e9b7c4b